requirements.txt

```diff
@@ -40,7 +40,7 @@ numpy>=1.18.4
 osmnx==0.15.0
 osmread @ git+https://github.com/dezhin/osmread.git@d8d3fe5edd15fdab9526ea7a100ee6c796315663#egg=osmread-0.2.dev0
 packaging==20.4
-pandas>=1.0.3
+pandas==1.3.3
```

Comment:
Upgrading pandas scared me slightly, but it didn't break any unit tests or notebooks.

Reply:
I changed it to >= quite recently, I wanted it to play nicely with MC (but don't need to in the end because SCOP went a different direction). I prefer the fixed versions I think